package.json

```diff
@@ -57,7 +58,8 @@
     "nebenan-helpers": "^4.3.0",
     "redux": "^4.0.5",
     "react-redux": "^7.2.2",
-    "react-router": "^5.2.0"
+    "react-router": "^5.2.0",
+    "lodash": "^4.17.15"
```

Comment:
If it was for me, I would have put it into `dependencies`. I wouldn't mind having the `lodash/debounce` function multiple times in the bundle because I don't want to fiddle around with peerDependencies

Reply:
50/50 about the same, we have lodash almost everywhere. lets go with peer dep